### PR TITLE
Make AWS region configurable

### PR DESCRIPTION
**Pull request by fbrnc**
Currently the AWS region is hardcode to us-east-1 (for Amazon SES). This PR introduces a new configuration option that let's you select your region.

### CODE REVIEW

File: app/code/local/Aschroder/SMTPPro/Model/System/Config/Source/Smtp/Awsregions.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            'us-east-1' => Mage::helper('adminhtml')->__('US East (N. Virginia)'),
+            'us-west-2' => Mage::helper('adminhtml')->__('US West (Oregon)'),
+            'us-west-1' => Mage::helper('adminhtml')->__('US West (N. California)'),
+            'eu-west-1' => Mage::helper('adminhtml')->__('EU (Ireland)'),
+            'eu-central-1' => Mage::helper('adminhtml')->__('EU (Frankfurt)'),
+            'ap-southeast-1' => Mage::helper('adminhtml')->__('Asia Pacific (Singapore)'),
+            'ap-northeast-1' => Mage::helper('adminhtml')->__('Asia Pacific (Tokyo)'),
+            'ap-southeast-2' => Mage::helper('adminhtml')->__('Asia Pacific (Sydney)'),
+            'ap-northeast-2' => Mage::helper('adminhtml')->__('Asia Pacific (Seoul)'),
+            'sa-east-1' => Mage::helper('adminhtml')->__('South America (Sao Paulo)')
```
Comment (aschroder) [inline app/code/local/Aschroder/SMTPPro/Model/System/Config/Source/Smtp/Awsregions.php]: Thanks for the PR! It looks good and happy to merge in next release. One thing though - are you sure all these regions are valid for SES? 

I thought only 3 were: US east/west and EU
http://docs.aws.amazon.com/ses/latest/DeveloperGuide/regions.html
(but I have not tested the others)

Comment (fbrnc) [inline app/code/local/Aschroder/SMTPPro/Model/System/Config/Source/Smtp/Awsregions.php]: Good point. Updated PR will follow in a couple of minutes...

**Comment by slikk66** [comment]
ran into this issue on client merge tonight.. what's the holdup on this merge? not everyone uses east-1 guys

**Comment by tschifftner** [comment]
Use the repo from aoepeople
Dan B <notifications@github.com> schrieb am Fr. 9. Juni 2017 um 08:48:

**Comment by sandermangel** [comment]
Hi @fbrnc  thank you for your contribution. This will be included in the next release

**Pull request merged**